cluster/v1/types.go

```diff
@@ -53,6 +53,11 @@ type SpokeClusterSpec struct {
 	// deleted.
 	// +required
 	HubAcceptsClient bool `json:"hubAcceptsClient" protobuf:"bytes,2,opt,name=hubAcceptsClient"`
+
+	// LeaseDurationSeconds is used to coordinate the lease update time of spoke agents.
+	// If it is empty, the spoke agent will update its lease per 1 minute
```

Comment:
If you allow both nil and zero you have to explain the difference between them.  If there is no difference, don't use a pointer.

For this particular field, I don't think I see value in allowing a zero.